main.go

```diff
@@ -1,12 +1,28 @@
 package main
 
 import (
+	"flag"
+	"log"
+
 	"github.com/hashicorp/terraform/plugin"
+	"github.com/hashicorp/terraform/terraform"
 	"github.com/sourcegraph/terraform-provider-site24x7/site24x7"
+	"github.com/sourcegraph/terraform-provider-site24x7/site24x7/oauth"
 )
 
+var oauthFile = flag.String("oauth_file", "", "path to the oauth_file")
```

Comment:
I don't know... just that using underscore in flags seems weird in Go:

```suggestion
var oauthFile = flag.String("oauth-file", "", "path to the OAuth file")
```

Reply:
done